build.sbt

```diff
@@ -571,7 +571,15 @@ def publishBinaryMtags =
     .in(interfaces)
     .dependsOn(
       publishAllMtags(
-        List(V.scala211, V.sbtScala, V.scala212, V.scala213, V.scala3)
+        List(
+          V.scala211,
+          V.sbtScala,
+          V.scala212,
+          // TODO https://github.com/scalameta/metals/issues/2392
```

Comment:
Can we just bump the version of Ammonite right away to not have to worry about this? It looks like it's available for 2.12.13 -> `2.3.8-32-64308dc3`. Or you you purposefully not including that?

Reply:
So turns out there was a regression, but once https://github.com/lihaoyi/Ammonite/pull/1146 is released we can upgrade.

Reply:
https://media.giphy.com/media/H3qWY13eGZUomKxOs6/giphy.mp4

Regressions this week...